webui/src/pages/repositories/repository/fileRenderers/duckdb.tsx

```diff
@@ -43,7 +43,7 @@ async function getDuckDB(): Promise<duckdb.AsyncDuckDB> {
 // taken from @duckdb/duckdb-wasm/dist/types/src/bindings/tokens.d.ts
 // which, unfortunately, we cannot import.
 const DUCKDB_STRING_CONSTANT = 2;
-const LAKEFS_URI_PATTERN = /^(['"]?)(lakefs:\/\/(.*))(['"])$/;
+const LAKEFS_URI_PATTERN = /^(['"]?)(lakefs:\/\/(.*))(['"])\s*$/;
```

Comment:
can we just drop the $ from the end of the patter? unless a full match is required only the second group match is what we after.

Reply:
Wanted to keep the token structure explicit to not accidentally capture something beyond the token. whitespaces are the only ignored characters so I prefer to add them explicitly